### PR TITLE
file: add doc on how to convert epoch seconds to datetime

### PR DESCRIPTION
**Pull request by jborean93**
##### SUMMARY
Add an example of how to convert a stat time output, or seconds since epoch to the datetime format expected by the file module.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
file

### CODE REVIEW

File: lib/ansible/modules/files/file.py [hunk 1]
```diff
@@ -181,6 +181,12 @@
     state: file
     modification_time: now
     access_time: now
+
+- name: Set access time based on seconds from epoch value
+  file:
+    path: /etc/another_file
+    state: file
+    access_time: '{{ "%Y%m%d%H%M.%S" | strftime(stat_var.stat.atime) }}'
```
Comment (bcoca) [inline lib/ansible/modules/files/file.py:189]: I don't think your strftime template is correct

Comment (jborean93) [reply lib/ansible/modules/files/file.py:189]: Hmm it worked for me, what part do you think isn't correct?

Comment (bcoca) [reply lib/ansible/modules/files/file.py:189]: i always used '%s' .. won't the above print yearmonthdayhourminute. seconds?

also i think we did strftime wrong, i would expect `dateortime|strftime('format')`

Comment (bcoca) [reply lib/ansible/modules/files/file.py:189]: ~@jborean93 i think you are doing the opposite transformation, you are turning 'seconds/epoch' into a date string~ 

nvmd, its opposite of what user wanted to implement, but how we can get 'epoch' to work with current module ... took me a bit to square the circle .. moving on ... nothing to see...

**Comment by fizista** [comment]
In this way it is not possible to set the time with greater accuracy than a second.  The strftime filter returns values with an accuracy of one second. The os.utime function is able to set the time with an accuracy of microseconds.

My previous patch allowed just such modifications.

**Pull request merged**